crypto/rsa/rsa_sp800_56b_gen.c

```diff
@@ -147,11 +147,15 @@ int ossl_rsa_fips186_4_gen_prob_primes(RSA *rsa, RSA_ACVP_TEST *test,
     ret = 1;
 err:
     /* Zeroize any internally generated values that are not returned */
```

Comment:
I guess this comment is out of date.

Reply:
the comment is still valid for `Xp0`, `Xq0` and `tmp`. Perhaps newly added code should come with comment saying `clear p and q in RSA structure too on failure`. I decided not to add more comments. IMO the code here explain itself.